src/js_native_api.h

```diff
@@ -450,6 +450,12 @@ napi_create_typedarray(napi_env env,
                        size_t byte_offset,
                        napi_value* result);
 NAPI_EXTERN napi_status NAPI_CDECL
+napi_create_buffer_from_arraybuffer(napi_env env,
```

Comment:
Please wrap this new method definition with `#ifdef NAPI_EXPERIMENTAL` so that it can be versioned.

Reply:
I Applied, thank you

Reply:
I got an error when trying to rewind, any comments? @legendecas

Reply:
Tests should also be wrapped with `NAPI_EXPERIMENTAL` defined. Please take a look at the error message.